oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/GraphQLQueryConfig.java

```diff
@@ -31,4 +31,5 @@ public class GraphQLQueryConfig extends ModuleConfig {
     private boolean enableLogTestTool;
     private int maxQueryComplexity = 1000;
     private boolean enableUpdateUITemplate = false;
+    private boolean enableOndemandLog = false;
```

Comment:
```suggestion
    private boolean enableOnDemandLog = false;
```
The most I saw about this is `on-demand` or `onDemand` as two words.